sdk/servicebus/service-bus/test/invalidParameters.spec.ts

```diff
@@ -24,277 +24,6 @@ describe("invalid parameters", () => {
-        TestClientType.PartitionedSubscription
-      );
-
-      subscriptionReceiverClient = await serviceBusClient.test.getPeekLockReceiver(entityNames);
-
-      // subscriptionRuleManager = serviceBusClient.test.addToCleanup(
-      //   serviceBusClient.getSubscriptionRuleManager(entityNames.topic!, entityNames.subscription!)
-      // );
-    });
-
-    after(() => {
-      return serviceBusClient.test.afterEach();
-    });
-
-    it("Peek: Invalid maxMessageCount for Subscription", async function(): Promise<void> {
-      const browsedMessages = await subscriptionReceiverClient.peekMessages(-100);
-      should.equal(browsedMessages.length, 0);
-    });
-
-    it("Peek: Wrong type maxMessageCount for Subscription", async function(): Promise<void> {
-      let caughtError: Error | undefined;
-      try {
-        // @ts-expect-error
-        await subscriptionReceiverClient.peekMessages("somestring");
-      } catch (error) {
-        caughtError = error;
-      }
-      should.equal(caughtError && caughtError.name, "TypeError");
-      should.equal(
-        caughtError && caughtError.message,
-        `The parameter "maxMessageCount" should be of type "number"`
-      );
-    });
-
-    it("PeekBySequenceNumber: Invalid maxMessageCount for Subscription", async function(): Promise<
-      void
-    > {
-      const browsedMessages = await subscriptionReceiverClient.peekMessages(-100, {
-        fromSequenceNumber: Long.ZERO
-      });
-      should.equal(browsedMessages.length, 0);
-    });
-
-    it("PeekBySequenceNumber: Wrong type maxMessageCount for Subscription", async function(): Promise<
-      void
-    > {
-      let caughtError: Error | undefined;
-      try {
-        // @ts-expect-error
-        await subscriptionReceiverClient.peekMessages("somestring", {
-          fromSequenceNumber: Long.ZERO
-        });
-      } catch (error) {
-        caughtError = error;
-      }
-      should.equal(caughtError && caughtError.name, "TypeError");
-      should.equal(
-        caughtError && caughtError.message,
-        `The parameter "maxMessageCount" should be of type "number"`
-      );
-    });
-
-    it("PeekBySequenceNumber: Wrong type fromSequenceNumber for Subscription", async function(): Promise<
-      void
-    > {
-      let caughtError: Error | undefined;
-      try {
-        await subscriptionReceiverClient.peekMessages(1, {
-          fromSequenceNumber: "somestring" as any
-        });
-      } catch (error) {
-        caughtError = error;
-      }
-      should.equal(caughtError && caughtError.name, "TypeError");
-      should.equal(
-        caughtError && caughtError.message,
-        `The parameter "fromSequenceNumber" should be of type "Long"`
-      );
-    });
-
-    // it("AddRule: Missing ruleName", async function(): Promise<void> {
-    //   let caughtError: Error | undefined;
-    //   try {
-    //     await subscriptionRuleManager.addRule(undefined as any, undefined as any);
```

Comment:
Then... Are we removing the source code for subscriptionRuleManager too?

Reply:
I am doing that in another PR that I already have put out. But it makes sense to include that change here. Will push a commit

Reply:
Created a separate PR as it turned out there are quite a few files that needed the clean up. See #9952

Reply:
#9952 is merged and this PR is updated